extensions/2.0/Khronos/KHR_instancing/extensions/2.0/Khronos/KHR_instancing/README.md

```diff
@@ -0,0 +1,64 @@
+# KHR\_instancing
+
+## Khronos 3D Formats Working Group
+
+* Alexey Knyazev
+* Don McCurdy
+* Ed Mackey
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+* Romain Guy
+* Tobias Haeussler
+* John Cooke
```

Comment:
Fine to just leave this list as a "John Cooke + TODO" for now, we usually fill it in at the end.

Reply:
oops.